compiler/src/iree/compiler/Dialect/Flow/Transforms/test/conv1x1_to_matmul.mlir

```diff
@@ -1,28 +1,29 @@
 // RUN: iree-opt --split-input-file -iree-flow-convert-conv2d-1x1-to-matmul %s | FileCheck %s
 
-func.func @conv_2d_1x1(%input: tensor<1x4x5x2xf32>, %filter: tensor<1x1x2x7xf32>) -> tensor<1x4x5x7xf32> {
+func.func @conv_nhwc_2d_1x1(%input: tensor<1x4x5x2xf32>, %filter: tensor<1x1x2x7xf32>) -> tensor<1x4x5x7xf32> {
```

Comment:
Nit: just `nhwc_conv_2d`. `1x1` is not useful here given the whole file is testing 1x1 filter cases.

Reply:
done